content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+described on this page.
+
+## Use the API Token to Authorize Access 
+
+Next, you can use the API Token to authorize access to our other endpoints. Take
+the API Token that you [generated](#create-an-api-token-in-the-cobalt-ui). Substitute that value for `YOUR-PERSONAL-API-TOKEN`:
+
+```
+curl https://api.cobalt.io/orgs \
+     -H "Accept: application/vnd.cobalt.v2+json" \
+     -H "Authorization: Bearer YOUR-PERSONAL-API-TOKEN" \
+     | jq .
+```
+
+{{%expand "Review sample output." %}}
+You should see output similar to:
+
+```
+{
+  "pagination": {
+    "next_page": null,
+    "prev_page": null
+  },
+  "data": [
+    {
+      "resource": {
+        "id": "some_id",
+        "name": "Saxophone - Alto",
+        "token": "YOUR-V2-ORGANIZATION-TOKEN"
+      },
+      "links": {
+        "ui": {
+          "url": "https://api.cobalt.io/links/not_a_jwt_token"
+        }
+      }
+    }
+  ]
+}
+```
+{{% /expand %}}
+  
+From the output, save the value for `token` as your organization token.
+In our API documentation, you'll see this as `YOUR-V2-ORGANIZATION-TOKEN`.
+
+For more information, see our API reference documentation on the
+[organizations](https://docs.cobalt.io/v2/#organizations) `orgs` endpoint.
+
+## Create an Asset
+
+Now that you have the following information:
+
+- `YOUR-PERSONAL-API-TOKEN`
+- `YOUR-V2-ORGANIZATION-TOKEN` 
+
+You can create an [asset](../getting-started/glossary/#asset) with the following REST call:
+
+```
+curl -X POST "https://api.cobalt.io/assets" \
+  -H 'Accept: application/vnd.cobalt.v2+json' \
+  -H 'Authorization: Bearer YOUR-PERSONAL-API-TOKEN' \
+  -H 'Content-Type: application/vnd.cobalt.v2+json' \
+  -H 'Idempotency-Key: A-UNIQUE-IDENTIFIER-TO-PREVENT-UNINTENTIONAL-DUPLICATION' \
+  -H 'X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN' \
+  --data '{
+            "title": "Test Asset",
+            "description": "How to describe the asset to our pentesters",
+            "asset_type": "web"
+          }' \
+  -v
+```
+
+For more information on each parameter, see our API Reference documentation on
+how to [Create an Asset](https://docs.cobalt.io/v2/#create-an-asset).
+
+The command we use includes a `-v`, which sets up output in verbose mode. The
+command works without it; however, you would see no response from the REST call.
+
+When you review the output of the REST call with the `-v`, look for the line
+with `HTTP/2`. You'll see one of the following lines:
+<!-- The output is associated with a `201` message, which doesn't include
+results, which is why I recommend a `-v` -->
+
+| Message    | Meaning          |
+|------------|------------------|
+| HTTP/2 201 | Asset created    |
+| HTTP/2 401 | No asset created. Check the value of your API Token|
+| HTTP/2 404 | No asset created. Check the value of YOUR-V2-ORGANIZATION-TOKEN|
+| HTTP/2 409 | No asset created |
+
+<!-- Maybe this table really belongs in our API reference, next to
+https://docs.cobalt.io/v2/#errors?  -->
```

Comment:
I'm not sure we need to include error codes in the guide. The OpenAPI specification contains more error codes, and  it's not easy to test all of them to provide a correct and meaningful message. I think the code returned for a successful request is sufficient.

Reply:
Point well taken. I might set up a link.

Reply:
see current lines 117, 118

Reply:
Looks good.